### PR TITLE
chore(image_decoder): remove unused codes

### PR DESCRIPTION
**Pull request by W-Mai**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

remove unused codes

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

File: src/draw/lv_image_decoder.c [hunk 1]
```diff
@@ -117,8 +117,8 @@ lv_result_t lv_image_decoder_open(lv_image_decoder_dsc_t * dsc, const void * src
     dsc->src_type = src_type;
 
     if(dsc->src_type == LV_IMAGE_SRC_FILE) {
-        size_t fnlen = lv_strlen(src);
-        dsc->src = lv_malloc(fnlen + 1);
+        size_t fn_len = lv_strlen(src);
```
Comment (XuNeo) [inline src/draw/lv_image_decoder.c]: dsc->src = lv_strdup(src)

Comment (W-Mai) [reply src/draw/lv_image_decoder.c]: great

**Review by kisvegabor — DISMISSED** [review]

**Comment by kisvegabor** [comment]
I've [temporary disabled the LOGs](https://github.com/lvgl/lv_binding_micropython/commit/a664c4ebdecaa2d6e718f1de81fc0d5bf1e14b16) in the binding until we find a solution. cc @PGNetHun

**Comment by kisvegabor** [comment]
https://github.com/lvgl/lvgl/pull/5395 is still needed.

**Comment by kisvegabor** [comment]
The CI should pass now, please rebase on `master`.

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**